Source/WebCore/platform/graphics/gstreamer/mse/MediaSourcePrivateGStreamer.h

```diff
@@ -64,6 +64,7 @@ class MediaSourcePrivateGStreamer final : public MediaSourcePrivate
 
     void durationChanged(const MediaTime&) override;
     void markEndOfStream(EndOfStreamStatus) override;
+    void unmarkEndOfStream() override;
```

Comment:
I wonder if we should be marking all these methods as final instead of override. Now now.